extensions/machine-learning/src/views/models/modelSourcesComponent.ts

```diff
@@ -110,29 +91,99 @@ export class ModelSourcesComponent extends ModelViewBase implements IDataCompone
 					break;
 			}
 		});
-		this._flexContainer = modelBuilder.flexContainer()
-			.withLayout({
-				flexFlow: 'row',
-				justifyContent: 'space-between'
-			}).withItems(components).component();
+		let radioCardGroup = modelBuilder.radioCardGroup()
+			.withProperties({
+				cards: components,
```

Comment:
please check with ctrl + and - that this looks okay?

Reply:
verified it look ok